src/seedu/addressbook/data/person/Person.java

```diff
@@ -84,4 +84,15 @@ public String toString() {
         return getAsTextShowAll();
     }
 
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    public String getPrintableString(Printable... printables) {
+        String result = "";
+        for (Printable p: printables) {
```

Comment:
Use a StringJoiner for string concatenation in a loop for better readability and performance.